app/src/main/res/values/strings.xml

```diff
@@ -620,6 +622,7 @@
     </string-array>
 
     <string-array name="languages_value" translatable="false">
+        <item>System_Language</item>
```

Comment:
do we really need to add this option if we are changing the language dynamically? I mean the moment we set our mobile to follow a particular language it should be reflected in our app at that very instance.

Reply:
Yes, we need it because if we are dependent on the System's Language then having a System Language value will be helpful. And sometimes if the System's Language is not supported in our App then English is made as Default. I have attached the videos for it also on the top.

This is also similar to the app theme preference on System Theme, Dark Theme, and Light Theme.
<img src="https://github.com/openMF/mifos-mobile/assets/73953395/c91397f4-0df3-46b5-90c0-75c98df78135" height=450/>